docs/Deprecating-Disabling-and-Removing-Formulae.md

```diff
@@ -18,19 +18,21 @@ A formula should be deprecated to indicate to users that the formula should not
 
 The most common reasons for deprecation are when the upstream project is deprecated, unmaintained, or archived.
 
-Formulae with dependents may be deprecated only if at least one of the following are true:
+Formulae should only be be deprecated if at least one of the following are true:
 
-- its dependents are all deprecated
 - the formula does not build on any of our supported macOS versions and on Linux
 - the formula has outstanding CVEs
+- the formula has [zero installs in the last 90 days](https://formulae.brew.sh/analytics/install/90d/)
```

Comment:
We have a bunch of reasons listed under "Deprecate and Disable Reasons" that don't match these, e.g. upstream EOL.

Reply:
@Bo98 Can you suggest a wording change if you consider this blocking, I have no idea what you're asking for here.

Reply:
GitHub won't let me but I guess the main one to add would be

```
- the software installed has been discontinued or abandoned upstream
```

I think all of the existing reason symbols (`:unsupported`, `:unmaintained`, `:repo_archived`, `: deprecated_upstream`, ...) really boil down to that as they all are just rewordings of the exact same core reason.

`:checksum_mismatch` is a bit of an odd one. Not sure if we actually deprecate/disable formulae for that?